.travis.yml

```diff
@@ -30,3 +30,20 @@ script:
 after_success:
 - sbt ++$TRAVIS_SCALA_VERSION orgAfterCISuccess
 - bash <(curl -s https://codecov.io/bash)
+
+cache:
+  directories:
+  - $HOME/.sbt/0.13
+  - $HOME/.sbt/boot/scala*
+  - $HOME/.sbt/cache
+  - $HOME/.sbt/launchers
+  - $HOME/.ivy2
```

Comment:
@juanpedromoreno From what it says  [in the Travis documentation](https://docs.travis-ci.com/user/caching/#Things-not-to-cache), I wonder if caching the `ivy2` directory does much good. ¿Are there any noticed speedups over `coursier` alone?